vagrant/tutorial/Vagrantfile

```diff
@@ -10,35 +10,10 @@ unless Vagrant.has_plugin?("vagrant-reload")
   raise "vagrant-reload plugin is not installed"
 end
 
-=begin
-  The following error may occur when running `vagrant up`, because
-  of a problem with some VirtualBox versions.
-
-    # Failed to mount folders in Linux guest. This is usually because
-    # the "vboxsf" file system is not available. Please verify that
-    # the guest additions are properly installed in the guest and
-    # can work properly. The command you attempted was:
-    #
-    # mount -t vboxsf -o uid=`id -u vagrant`,gid=`getent group vagrant | cut -d: -f3` vagrant /vagrant
-    # mount -t vboxsf -o uid=`id -u vagrant`,gid=`id -g vagrant` vagrant /vagrant
-    #
-    # The error output from the last command was:
-    #
-    # /sbin/mount.vboxsf: mounting failed with the error: No such device
-
-  To help avoid this potential issue, the vagrant-vbguest plugin is required.
-  If it remains, try updating VirtualBox.
-  Known working versions include Vagrant 1.7.2 and VirtualBox 4.3.24r98716.
-=end
-unless Vagrant.has_plugin?("vagrant-vbguest")
-  raise "vagrant-vbguest plugin is not installed"
```

Comment:
So it sounds like it *is* safe to remove this from the tutorial Vagrantfile and that the plugin *is* still needed in the dev  Vagrantfile.

The build for this branch still shows 

```
Copy iso file /usr/share/virtualbox/VBoxGuestAdditions.iso into the box /tmp/VBoxGuestAdditions.iso
mount: /dev/loop0 is write-protected, mounting read-only
Installing Virtualbox Guest Additions 4.3.20 - guest version is 4.3.26
Verifying archive integrity... All good.
Uncompressing VirtualBox 4.3.20 Guest Additions for Linux............
VirtualBox Guest Additions installer
Removing installed version 4.3.26 of VirtualBox Guest Additions...
Copying additional installer modules ...
Installing additional modules ...
Removing existing VirtualBox non-DKMS kernel modules[  OK  ]
Building the VirtualBox Guest Additions kernel modules
Building the main Guest Additions module[FAILED]
(Look at /var/log/vboxadd-install.log to find out what went wrong)
Doing non-kernel setup of the Guest Additions[  OK  ]
Installing the Window System drivers
Could not find the X.Org or XFree86 Window System, skipping.
An error occurred during installation of VirtualBox Guest Additions 4.3.20. Some functionality may not work as intended.
In most cases it is OK that the "Window System drivers" installation failed.
Restarting VM to apply changes...
```

 * http://build.clusterhq.com/builders/flocker%2Fvagrant%2Fbuild%2Ftutorial/builds/119/steps/build-base-box/logs/stdio

Is that what we expect?